src/Adapter/Kpi/ShoppingCartTotalKpi.php

```diff
@@ -57,9 +72,9 @@ public function render()
         $helper->color = 'color1';
         $helper->title = $translator->trans('Total Cart', [], 'Admin.Orderscustomers.Feature');
         $helper->subtitle = $translator->trans('Cart #%ID%', ['%ID%' => $cart->id], 'Admin.Orderscustomers.Feature');
-        $helper->value = Tools::displayPrice(
+        $helper->value = $this->locale->formatPrice(
             $this->getCartTotalPrice($cart),
-            (int) $cart->id_currency
+            \Currency::getIsoCodeById((int) $cart->id_currency)
```

Comment:
Please add a `use Currency;` instead of the `\Currency` notation

Reply:
Fixed